packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/albums/AddAssetsToAlbum.kt

```diff
@@ -43,11 +43,6 @@ internal class AddAssetsToAlbum(
         albumCursor.moveToNext()
         val filePathColumnIndex = albumCursor.getColumnIndex(MediaStore.Images.Media.DATA)
         val fileInAlbum = File(albumCursor.getString(filePathColumnIndex))
-
-        // Media store table can be corrupted. Extra check won't harm anyone.
-        if (!fileInAlbum.isFile) {
-          throw MediaLibraryException()
-        }
```

Comment:
Maybe we can keep a check like this:
```        
// Media store table can be corrupted. Extra check won't harm anyone.
if (!fileInAlbum.isFile && !fileInAlbum.isDirectory) {
  throw MediaLibraryException()
}
 ```
 This should be safer around symbolic links, files with no read permission etc and should still work with subdirectories.

Reply:
Good idea, thanks.